layouts/episode/single.html

```diff
@@ -1,8 +1,18 @@
 {{ define "main" }}
 
   <div class="row">
-    <div class="col">
-      <h1>{{ title .Title }}</h1>
+    <div class="col">      
+      {{ if isset .Site.Params "episode_number_style"}}
```

Comment:
Might want to check for both if `episode_number_style` is set as well as `.Params.episode` since if the episode doesn't have one set, it will error (it's possible that older episodes might not have them set, etc etc?)

Reply:
Ah, good catch. So refactoring all instances of the conditional to be something like {{ if and (isset .Site.Params "episode_number_style") (isset .Params "episode") {{

Reply:
Making a slight alteration - Will use if and (isset .Site.Params "episode_number_style") (.Params.episode) }}.  That then accounts for the empty string scenario. Will also add an else to that initial if, to cover the title on its own, and then we're wrapped up.